http/server.ts

```diff
@@ -122,6 +126,26 @@ export class ServerRequest {
   conn: Conn;
   r: BufReader;
   w: BufWriter;
+  private _cookie: Cookie;
```

Comment:
The last question about the cache of cookie:

Headers got the interface to delete the field.

Caching is to improve performance, but this may result in inconsistent results

```typescript
import { serve } from 'https://deno.land/x/std/http/server.ts';
import { assertEquals } from 'https://deno.land/x/std/testing/asserts.ts';

const s = serve('0.0.0.0:8000');

async function main() {
  for await (const req of s) {
    // read it first
    const cookie1 = req.cookie;
    // delete cookie
    req.headers.delete('cookie');
    // read again. but it will read from cache
    const cookie2 = req.cookie;

    // should not equals
    assertEquals(cookie1, cookie2);

    req.respond({ body: new TextEncoder().encode('Hello World\n') });
  }
}

main();
```

How do you see? @ry

Reply:
Server never respond with `Cookie` header but with only `Set-Cookie`. The `Cookie` header is only sent by client so this case will never appears. I think?

Also that's the case when we'd prefer to use `cookie(reload: boolean)` instead for `get cookie` for example.